### PR TITLE
disable jhsingle keepalive behavior for jupyterhub idle culler

### PR DESCRIPTION
**Pull request by tylergraff**
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [X] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Comment by tylergraff** [comment]
refs https://github.com/nebari-dev/jhub-apps/issues/136

File: jhub_apps/spawner/command.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def get_substituted_args(self, **kwargs):
         TString("$python_exec"),
         "-m",
         "jhsingle_native_proxy.main",
-        TString("--authtype=$authtype"),
+        TString("--authtype=$authtype --no-force-alive"),
```
Comment (aktech) [inline jhub_apps/spawner/command.py:57]: I believe the failing tests should work if you put this in a separate line:

```python
 TString("--authtype=$authtype"),
"--no-force-alive",
```

Comment (tylergraff) [reply jhub_apps/spawner/command.py]: :man_facepalming: 
thanks!

**Review by aktech — APPROVED** [review]

**Comment by aktech** [comment]
This looks good to me, will handle https://github.com/nebari-dev/jhub-apps/issues/139 separately.

**Pull request merged**